dask/array/tests/test_fft.py

```diff
@@ -31,80 +31,52 @@ def test_cant_fft_chunked_axis():
         fft(bad_darr, axis=0)
 
 
-def test_fft():
-    assert_eq(fft(darr), np.fft.fft(nparr))
-    assert_eq(fft(darr2, axis=0), np.fft.fft(nparr, axis=0))
-
-
-def test_fft_n_kwarg():
-    assert_eq(fft(darr, 5), np.fft.fft(nparr, 5))
-    assert_eq(fft(darr, 13), np.fft.fft(nparr, 13))
-    assert_eq(fft(darr2, 5, axis=0), np.fft.fft(nparr, 5, axis=0))
-    assert_eq(fft(darr2, 13, axis=0), np.fft.fft(nparr, 13, axis=0))
-
-
-def test_ifft():
-    assert_eq(ifft(darr), np.fft.ifft(nparr))
-    assert_eq(ifft(darr2, axis=0), np.fft.ifft(nparr, axis=0))
-
-
-def test_ifft_n_kwarg():
-    assert_eq(ifft(darr, 5), np.fft.ifft(nparr, 5))
-    assert_eq(ifft(darr, 13), np.fft.ifft(nparr, 13))
-    assert_eq(ifft(darr2, 5, axis=0), np.fft.ifft(nparr, 5, axis=0))
-    assert_eq(ifft(darr2, 13, axis=0), np.fft.ifft(nparr, 13, axis=0))
-
-
-def test_rfft():
-    assert_eq(rfft(darr), np.fft.rfft(nparr))
-    assert_eq(rfft(darr2, axis=0), np.fft.rfft(nparr, axis=0))
-
-
-def test_rfft_n_kwarg():
-    assert_eq(rfft(darr, 5), np.fft.rfft(nparr, 5))
-    assert_eq(rfft(darr, 13), np.fft.rfft(nparr, 13))
-    assert_eq(rfft(darr2, 5, axis=0), np.fft.rfft(nparr, 5, axis=0))
-    assert_eq(rfft(darr2, 13, axis=0), np.fft.rfft(nparr, 13, axis=0))
-    assert_eq(rfft(darr2, 12, axis=0), np.fft.rfft(nparr, 12, axis=0))
-
-
-def test_irfft():
-    assert_eq(irfft(darr), np.fft.irfft(nparr))
-    assert_eq(irfft(darr2, axis=0), np.fft.irfft(nparr, axis=0))
-
-
-def test_irfft_n_kwarg():
-    assert_eq(irfft(darr, 5), np.fft.irfft(nparr, 5))
-    assert_eq(irfft(darr, 13), np.fft.irfft(nparr, 13))
-    assert_eq(irfft(darr2, 5, axis=0), np.fft.irfft(nparr, 5, axis=0))
-    assert_eq(irfft(darr2, 13, axis=0), np.fft.irfft(nparr, 13, axis=0))
-    assert_eq(irfft(darr2, 12, axis=0), np.fft.irfft(nparr, 12, axis=0))
-
-
-def test_hfft():
-    assert_eq(hfft(darr), np.fft.hfft(nparr))
-    assert_eq(hfft(darr2, axis=0), np.fft.hfft(nparr, axis=0))
-
-
-def test_hfft_nkwarg():
-    assert_eq(hfft(darr, 5), np.fft.hfft(nparr, 5))
-    assert_eq(hfft(darr, 13), np.fft.hfft(nparr, 13))
-    assert_eq(hfft(darr2, 5, axis=0), np.fft.hfft(nparr, 5, axis=0))
-    assert_eq(hfft(darr2, 13, axis=0), np.fft.hfft(nparr, 13, axis=0))
-    assert_eq(hfft(darr2, 12, axis=0), np.fft.hfft(nparr, 12, axis=0))
-
+@pytest.mark.parametrize(
```

Comment:
Sorry I'm very new to pytest (used nose more). I think I have a vague idea of what this `fixture` thing is doing and did try to read the docs, but could you please help me understand what is happening with this? Does it just let me delete the `parameterize` decorator from the tests if I add those 3 lines to this file? Is it doing anything more than that?